.travis.yml

```diff
@@ -1,13 +1,13 @@
 language: node_js
 node_js:
   - "lts/carbon"
-  - "10.13.0"
-  - "10.15.0"
```

Comment:
I think that we should leave the build for 10.15.0 in as well at the moment since that's the highest version that we are supporting.